05-JWT-Basics/preferred/controllers/main.js

```diff
@@ -0,0 +1,26 @@
+const jwt = require('jsonwebtoken');
+
+const logon = async (req, res) => {
+    const { username, password } = req.body;
```

Comment:
Minor issue. The assignment specified "name" and "password" as the attribute names.
username should be changed to name here.

Reply:
@olearyma thanks! Made a commit with the required changes.